.vscode/settings.json

```diff
@@ -0,0 +1,5 @@
+{
+    "gopls": {
+        "build.buildFlags": ["-tags=integration_test"]
```

Comment:
This feels like a hack, as it's a workspace-wide setting that's intended to only affect a sub-folder within that workspace. IMO a more proper way to achieve this is to move `.vscode/settings.json` to `integration_test/` and then open `integration_test/` as a root folder in vscode. Unfortunately, since https://github.com/Microsoft/vscode/issues/32693 isn't resolved yet, this means you would need a second editor window open.

Reply:
I disagree that it's a hack. For one thing, the `integration_test` build tag is supposed to apply to the entire project when it's applied for a build so I think it's fair to apply it to the whole project's settings. As well, what you're suggesting would track if `integration_test` was a module on its own, but it's a sub-package of the overall `ops-agent` module. Opening `integration_test` in its own window would break, since no `go.mod` file would be detected. 

Now whether `integration_test` should be a module of its own is maybe worth digging into, in which case I'd certainly want to make this suggested change.

Reply:
Does adding -tags=integration_test disable gopls for most of the code in this repo? Since none of that code has that build tag, doesn't go refuse to build it? Not sure.

Reply:
True, I forgot about modules. I might've overstated the issue then. It's an improvement at any rate and I don't have a better suggestion at this point, except for wishing that gopls wasn't so strict :(